src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -34,16 +34,19 @@ def __init__(
         security_enabled: bool,
         component_test_config: str,
         test_recorder: TestRecorder,
+        xport: int = 9200
     ) -> None:
         super().__init__(
             work_dir,
             component_name,
             component_test_config,
             security_enabled,
             additional_cluster_config,
-            test_recorder.local_cluster_logs
+            test_recorder.local_cluster_logs,
+            xport
```

Comment:
Can we have a better naming convention than `xport`?

Reply:
Renamed